CMakeLists.txt

```diff
@@ -15,12 +15,10 @@ option( EXIV2_ENABLE_PNG              "Build with png support (requires libz)"
 cmake_dependent_option(EXIV2_ENABLE_NLS "Build native language support (requires gettext)" ON "UNIX" OFF)
 option( EXIV2_ENABLE_PRINTUCS2        "Build with Printucs2"                                  ON  )
 option( EXIV2_ENABLE_LENSDATA         "Build including lens data"                             ON  )
-option( EXIV2_ENABLE_VIDEO            "Build video support into library"                      OFF )
 option( EXIV2_ENABLE_DYNAMIC_RUNTIME  "Use dynamic runtime (used for static libs)"            ON )
 option( EXIV2_ENABLE_WIN_UNICODE      "Use Unicode paths (wstring) on Windows"                OFF )
 option( EXIV2_ENABLE_WEBREADY         "Build webready support into library"                   OFF )
 option( EXIV2_ENABLE_CURL             "USE Libcurl for HttpIo (WEBREADY)"                     OFF )
-option( EXIV2_ENABLE_SSH              "USE Libssh for SshIo (WEBREADY)"                       OFF )
```

Comment:
There are many other occurrences of that variable in the CMake code and documentation. Could you remove the rest of them?

```
14:16 $ ag EXIV2_ENABLE_SSH
README-CONAN.md
608:$ cmake -DEXIV2_ENABLE_WEBREADY=ON -DEXIV2_ENABLE_CURL=ON -DEXIV2_ENABLE_SSH=ON ..

cmake/generateConfigFile.cmake
6:    set(EXV_USE_SSH   ${EXIV2_ENABLE_SSH})

cmake/printSummary.cmake
53:    OptionOutput( "USE Libssh for SshIo:               " EXIV2_ENABLE_SSH            )

cmake/findDependencies.cmake
22:    if( EXIV2_ENABLE_SSH )

src/CMakeLists.txt
110:    if( EXIV2_ENABLE_SSH )
170:    if( EXIV2_ENABLE_SSH )

CMakeLists.txt
23:option( EXIV2_ENABLE_SSH              "USE Libssh for SshIo (WEBREADY)"                       OFF )
```